code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -347,15 +347,15 @@ var/global/list/damage_icon_parts = list()
 	var/obj/item/organ/internal/eyes/eyes = get_organ(species.vision_organ || BP_EYES)
 	icon_key += istype(eyes) ? eyes.eye_colour : COLOR_BLACK
 
-	for(var/organ_tag in species.has_limbs)
+	for(var/organ_tag in species.has_limbs) // This means non-species limb slots won't draw, todo
```

Comment:
Good catch!

Reply:
Changed this to use the global limb flag list instead.